test/ansible/molecule/kind/create.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Create
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: Create test kind cluster
+      command: kind create cluster --name osdk-test --kubeconfig {{ kubeconfig }}
```

Comment:
Non-blocking comment (followup maybe?)

Could we make the cluster name more descriptive, like osdk-molecule-test? Its hard to remember which cluster is osdk-test and which is operator-sdk-e2e

Reply:
This file is the default scaffold done by the tool. 
we can raise an issue for we track that, but then we need to change the default scaffold.